### PR TITLE
QF-1678 Fixed logging for results which are now reported by API

### PR DESCRIPTION
**Pull request by ChrisJWoodcock**

### CODE REVIEW

File: src/SFA.DAS.ApprenticeFeedback.Jobs/Functions/ApprenticeFeedbackSummaryFunction.cs [hunk 1]
```diff
@@ -5,24 +5,39 @@
...
+        {
+            _log = log;
+            _endpoint = endpoint;
+        }
 
         [FunctionName("ApprenticeFeedbackSummaryTimer")]
         public void ApprenticeFeedbackSummaryTimer(
             [TimerTrigger("%FunctionsOptions:ApprenticeFeedbackSummarySchedule%")] TimerInfo timer, 
             ExecutionContext executionContext, 
             ILogger logger)
```
Comment (Jordan-fujitsu) [inline src/SFA.DAS.ApprenticeFeedback.Jobs/Functions/ApprenticeFeedbackSummaryFunction.cs]: Does the `ILogger logger` param need to be here could `_log` variable be used instead?

File: src/SFA.DAS.ApprenticeFeedback.Jobs/Infrastructure/Api/IApprenticeFeedbackApi.cs [hunk 2]
```diff
@@ -31,6 +31,6 @@ public interface IApprenticeFeedbackApi
         Task<UpdateApprenticeFeedbackTargetResponse> UpdateFeedbackTarget([Body] UpdateApprenticeFeedbackTargetRequest apprenticeFeedbackTargetToUpdate);
 
         [Post("feedbacktransaction")]
-        Task<GetEmailTransactionsResponse> GenerateEmailTransactions();
+        Task<NullResponse> GenerateEmailTransactions();
```
Comment (Jordan-fujitsu) [inline src/SFA.DAS.ApprenticeFeedback.Jobs/Infrastructure/Api/IApprenticeFeedbackApi.cs]: Why does this need to be an empty class (`NullResponse`), could you use `null` instead or have a `Task` they way it is for `GenerateFeedbackSummaries();`

File: src/SFA.DAS.ApprenticeFeedback.Jobs/Functions/ApprenticeFeedbackSummaryFunction.cs (hunk 1 above)
Comment (ChrisJWoodcock) [reply src/SFA.DAS.ApprenticeFeedback.Jobs/Functions/ApprenticeFeedbackSummaryFunction.cs]: Removed that

File: src/SFA.DAS.ApprenticeFeedback.Jobs/Infrastructure/Api/IApprenticeFeedbackApi.cs (hunk 2 above)
Comment (ChrisJWoodcock) [reply src/SFA.DAS.ApprenticeFeedback.Jobs/Infrastructure/Api/IApprenticeFeedbackApi.cs]: Removed the NullResponse it is needed in the Outer API but not needed to be passed to the jobs

**Review by Jordan-fujitsu — APPROVED** [review]

File: src/SFA.DAS.ApprenticeFeedback.Jobs/Functions/ApprenticeFeedbackSummaryFunction.cs (hunk 1 above)
Comment (ChrisJWoodcock) [reply src/SFA.DAS.ApprenticeFeedback.Jobs/Functions/ApprenticeFeedbackSummaryFunction.cs]: Fixed

**Review by divesh987 — APPROVED** [review]

**Pull request merged**